flow360/component/simulation/models/volume_models.py

```diff
@@ -0,0 +1,128 @@
+from typing import Optional
+
+import pydantic as pd
+
+from flow360.component.simulation.framework.base_model import Flow360BaseModel
+from flow360.component.simulation.models.material import Air, MaterialType
+from flow360.component.simulation.models.solver_numerics import (
+    NavierStokesSolver,
+    TransitionModelSolver,
+    TurbulenceModelSolverType,
+)
+
+
+class VolumeModelBase(Flow360BaseModel):
+    """
+    Base class for equation models
+
+    """
+
+    volumes: EntityList[Union[GenericVolume, str]] = pd.Field()
+    material: MaterialType = pd.Field()
+    initial_conditions: Optional[dict] = pd.Field(None)
+
+
+class FluidDynamicsAddOnBase(Flow360BaseModel):
+    volumes: EntityList[Union[GenericVolume, str]] = pd.Field()
+
+
+class FluidDynamics(VolumeModelBase):
+    """
+    General FluidDynamics volume model that contains all the common fields every fluid dynamics zone should have.
+    """
+
+    navier_stokes_solver: Optional[NavierStokesSolver] = pd.Field(None)
+    turbulence_model_solver: Optional[TurbulenceModelSolverType] = pd.Field(None)
```

Comment:
why defaults are none? We probably want defaults here

Reply:
Changed defaults. Why did we have `NoneSolver()` in flow360_params? What's the reason for using it instead of `None`?

Reply:
I think this was legacy reason. Solver expected turbulence having a type "None", not being equal to None. It applies defaults when solver is not provided. Also, there was some difficulties with heat solver, I remember. So we ended up implementing `NoneSolver`. But I am up for removing `NoneSolver` if we can do it.